enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -83,13 +84,18 @@ class SubsidyAccessPolicy(TimeStampedModel):
 
     history = HistoricalRecords()
 
-    @cached_property
-    def subsidy_client(self):
+    @classmethod
+    @functools.lru_cache(maxsize=None)
+    def get_subsidy_client(cls):
         """
         A request-cached EnterpriseSubsidyAPIClient instance.
         """
         return EnterpriseSubsidyAPIClient()
 
+    @property
+    def subsidy_client(self):
+        return self.get_subsidy_client()
+
```

Comment:
ah, I think you ran into some of the same mocking issues I did yesterday :)   See the comment in the `setUp()` method at this file for an idea on how to work around such issues: https://github.com/openedx/enterprise-access/pull/125/files#diff-9a1969f65a1bbadbe45014eafb5a9d6c2fa59d394f963c3023b9a374dc2919bd